templates/deployment.yaml

```diff
@@ -754,10 +754,10 @@ spec:
       - name: init-script
         configMap:
           name: {{ template "CGW.fullname" . }}-init-script
+          defaultMode: 0544
           items:
             - key: initScript.sh
               path: initScript.sh
-              defaultMode: 0544
```

Comment:
To use `mode` instead of `defaultMode` here would also be correct. However, the effect should be the same as setting the `defaultMode` for _all_ items as there is only one.